bridge/engine/ActiveMediaList.h

```diff
@@ -160,6 +168,7 @@ class ActiveMediaList
         int32_t _nonZeroLevelsShortWindow;
         float _maxRecentLevel;
         float _noiseLevel;
+        bool _isPtt;
```

Comment:
should we call this pushToTalk instead?
To avoid abbreviations that are confusing to .. anyone